annyang.js

```diff
@@ -462,9 +468,9 @@
      * });
      *
      * annyang.addCallback('resultMatch', function(userSaid, commandText, phrases) {
-     *   console.log(userSaid); // sample output: 'hello'
-     *   console.log(commandText); // sample output: 'hello (there)'
-     *   console.log(phrases); // sample output: ['hello', 'halo', 'yellow', 'polo', 'hello kitty']
+     *   logMessage(userSaid); // sample output: 'hello'
+     *   logMessage(commandText); // sample output: 'hello (there)'
+     *   logMessage(phrases); // sample output: ['hello', 'halo', 'yellow', 'polo', 'hello kitty']
```

Comment:
This three lines shouldn't be changed. This is sample code for the API docs that shows the user a sample callback he can use.

You can see it here: https://github.com/TalAter/annyang/tree/master/docs#examples-4